noark-extraction-validator/src/main/java/com/documaster/validator/validation/collector/ValidationResult.java

```diff
@@ -0,0 +1,143 @@
+/**
+ * Noark Extraction Validator
+ * Copyright (C) 2017, Documaster AS
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.documaster.validator.validation.collector;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.documaster.validator.storage.model.BaseItem;
+
+public class ValidationResult {
+
+	private String id;
+	private String title;
+	private String description;
+	private String groupName;
+
+	private List<BaseItem> information;
```

Comment:
Warnings and errors could potentially remain null and I did not want to instantiate lists unnecessarily. I see that with this micro-optimization the code is now less readable. Lists are now instantiated on declaration.